src/js/components/SearchPage.test.js

```diff
@@ -3,15 +3,13 @@ import { mount } from "enzyme"
 import { act } from "react-dom/test-utils"
 import { search } from "../lib/api"
 import { times } from "ramda"
-import {
-  INITIAL_FACET_STATE,
-  LR_TYPE_ALL
-} from "@mitodl/course-search-utils/dist/constants"
+import { INITIAL_FACET_STATE } from "@mitodl/course-search-utils/dist/constants"
 import { serializeSearchParams } from "@mitodl/course-search-utils/dist/url_utils"
 
 import SearchPage, { SEARCH_PAGE_SIZE } from "./SearchPage"
 
 import { makeCourseResult } from "../factories/search"
+import { LR_TYPE_COURSE, LR_TYPE_RESOURCEFILE } from "../lib/constants"
```

Comment:
one thing I was just thinking - it would be good to use the constants exported by `@mitodl/course-search-utils` I think, I'm working on adding that package to Open too, and once we have it there we can define these constants in only one place (instead of several, haha)